.commitlintrc.no-scope.yml

```diff
@@ -0,0 +1,34 @@
+rules:
```

Comment:
This file defines a set of rules that [commitlint](https://keisukeyamashita.github.io/commitlint-rs/) will enforce when no scope is found in the PR title.

Workaround for https://github.com/KeisukeYamashita/commitlint-rs/issues/355.